packages/twenty-front/src/modules/ui/input/components/IconPicker.tsx

```diff
@@ -29,6 +35,7 @@ export type IconPickerProps = {
   variant?: IconButtonVariant;
   className?: string;
   disableBlur?: boolean;
+  takeControlOfCustomIcon?: boolean;
```

Comment:
I don't think we need this additional props, we can already provide the selectedIconKey directly